foxtrot-core/src/main/java/com/flipkart/foxtrot/core/table/impl/DistributedTableMetadataManager.java

```diff
@@ -41,21 +73,80 @@
 public class DistributedTableMetadataManager implements TableMetadataManager {
     private static final Logger logger = LoggerFactory.getLogger(DistributedTableMetadataManager.class);
     public static final String DATA_MAP = "tablemetadatamap";
+    public static final String FIELD_MAP = "tablefieldmap";
     private final HazelcastConnection hazelcastConnection;
+    private final ElasticsearchConnection elasticsearchConnection;
+    private final ObjectMapper mapper;
     private IMap<String, Table> tableDataStore;
+    private IMap<String, TableFieldMapping> fieldDataCache;
 
     public DistributedTableMetadataManager(HazelcastConnection hazelcastConnection,
-                                           ElasticsearchConnection elasticsearchConnection) {
+                                           ElasticsearchConnection elasticsearchConnection,
+                                           ObjectMapper mapper) {
         this.hazelcastConnection = hazelcastConnection;
+        this.elasticsearchConnection = elasticsearchConnection;
+        this.mapper = mapper;
+
+        hazelcastConnection.getHazelcastConfig().getMapConfigs().put(DATA_MAP, tableMapConfig());
+        hazelcastConnection.getHazelcastConfig().getMapConfigs().put(FIELD_MAP, fieldMetaMapConfig());
+    }
+
+
+    private MapConfig tableMapConfig() {
+        MapConfig mapConfig = new MapConfig();
+        mapConfig.setReadBackupData(true);
+        mapConfig.setInMemoryFormat(InMemoryFormat.BINARY);
+        mapConfig.setTimeToLiveSeconds(10);
```

Comment:
@santanusinha @r0goyal It will cause cache to clear after 10 seconds and we won't be able to get the config for the table
hazelcastConnection.getHazelcastConfig().getMapConfigs().get(DATA_MAP) will return an empty config